packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -157,10 +157,32 @@ static inline BOOL IsHorizontalNavigationWithinZoneAction(FocusZoneAction action
 	return nil;
 }
 
+/// Bypass FocusZone if it's empty or has no focusable elements
+static BOOL ShouldSkipFocusZone(NSView *view)
+{
+    if([view isKindOfClass:[RCTFocusZone class]])
+    {
+        // FocusZone has no elements
+        if ([[view subviews] count] < 1)
```

Comment:
this isn't combinable with line 171. if there isn't a FirstKeyviewwithin the view, there must be no elements in the focuszone?

Reply:
That's correct. My rationale behind adding that check was for readability and avoid calling `GetFirstKeyViewWithin` on an empty FocusZone.

Reply:
We could move the "check is empty logic" inside `GetFirstKeyViewWithin` too. But I think that function should be relatively cheap since it will do nothing if there are no subviews.

Reply:
I went with adding the "is empty" check inside `GetFirstKeyViewWithin`. Thought it'd benefit every caller that way